bin/dsh

```diff
@@ -995,6 +995,7 @@ docker_machine_create ()
 			docker-machine create --driver=virtualbox \
 				--virtualbox-disk-size "$DEFAULT_MACHINE_VBOX_HDD" \
 				--virtualbox-memory "$DEFAULT_MACHINE_VBOX_RAM" \
+				--virtualbox-hostonly-cidr "$DRUDE_SUBNET" \
```

Comment:
Why do we need this? As far as I see vbox network is 192.168.99.1/24 it does not clash with 192.168.10.1